src/shaders/symbol_sdf.fragment.glsl

```diff
@@ -1,12 +1,19 @@
+#define SDF_PX 8.0
+#define BLUR_OFFSET 1.19
+#define HALO_OFFSET 6.0
+#define GAMMA 0.105/DEVICE_PIXEL_RATIO
```

Comment:
@lucaswoj I inlined a couple of them, but chose to leave `SDF_PX` and `EDGE_GAMMA` (my weak attempt at renaming `GAMMA`), because they are used in more than one place -- I did not want to inline them and then later lose track of the fact that both "8.0"s are supposed to mean the same thing.  Filed a follow up ticket though: https://github.com/mapbox/mapbox-gl-js/issues/4229